### PR TITLE
Implements #232 & #247

### PR DESCRIPTION
**Pull request by harsh242**
**What does this PR do?**
Implements #232 & #247

### CODE REVIEW

**Comment by harsh242** [comment]
@bhaskarSingh  can you take a look at how to solve the build fix.
It is happening because of a dependency in reacttour. 
I tried [this](https://www.gatsbyjs.com/docs/debugging-html-builds/#fixing-third-party-modules) as a solution found it via [this](https://github.com/gatsbyjs/gatsby/issues/11885), but couldn't hack my way out of it.

**Comment by bhaskarSingh** [comment]
@harsh242 as a workaround you can try using either loadable or react lazy/suspense component. Here's a example https://www.gatsbyjs.com/docs/using-client-side-only-packages/

**Comment by bhaskarSingh** [comment]
Screenshot 2021-03-17 at 9 07 20 PM
Screenshot 2021-03-17 at 9 07 26 PM 
Screenshot 2021-03-17 at 9 09 31 PM

The top and bottom of the page need a blackish background. Remove button which allows user go to academy & marketplace view. 

Also, sending mail feature also needs to be implemented. I suppose you already know this.

Preferably please create a new PR for fixing the #212 issue.

**Comment by bhaskarSingh** [comment]
> Should I reset this PR to 3eb4001 so that it's ready to be merged? @bhaskarSingh

Yes.

File: package.json [hunk 1]
```diff
@@ -66,17 +66,21 @@
     "react-colorful": "^4.4.4",
     "react-confetti": "^6.0.0",
     "react-dom": "^16.12.0",
+    "react-device-detect":"^1.17.0",
```
Comment (bhaskarSingh) [inline package.json:69]: @harsh242  This lib is not required anymore. Please remove it.

File: package.json [hunk 2]
```diff
@@ -66,17 +66,21 @@
...
     "react-loader-spinner": "^3.1.14",
     "react-responsive": "^8.0.3",
     "react-spring": "^8.0.27",
     "react-three-fiber": "^5.3.18",
     "react-tooltip": "^4.2.5",
     "react-transition-group": "^4.4.1",
     "react-typography": "^0.16.19",
+    "reactour":"^1.18.3",
     "react-use": "^15.3.8",
+    "styled-components":"^4.0.0",
```
Comment (bhaskarSingh) [inline package.json:83]: @harsh242 This lib is not being used anywhere. Also, remove it.

Comment (harsh242) [reply package.json:83]: [Reactour](https://github.com/elrumordelaluz/reactour) requires styled components

Comment (bhaskarSingh) [reply package.json:83]: Got it.

**Review by bhaskarSingh — APPROVED** [review]

**Pull request merged**